cilium-cli/Makefile

```
@@ -10,7 +10,7 @@ all: $(TARGET)
 .PHONY: all $(TARGET) local-release install clean test bench clean-tags tags
 
 
-BINDIR = /usr/local/bin
+BINDIR ?= /usr/local/bin
```

```suggestion
PREFIX=/usr/local
```

Because there is already `BINDIR?=$(PREFIX)/bin` in `Makefile.defs` that's included above, that now takes precedence, leading to `BINDIR` being `/usr/bin` by default instead of `/usr/local/bin` which should be the default value here for cilium-cli. 

A solution is to not set `BINDIR` in this Makefile (hence leaving it to be `BINDIR?=$(PREFIX)/bin`) and instead configure a different `PREFIX`. With this, you can control the install directory by setting `BINDIR`:
```sh
$ BINDIR=~/.local/bin make -C cilium-cli install
  [...]
  install -m 0755 -d /home/hadrien/.local/bin
  install -m 0755 cilium /home/hadrien/.local/bin
```

While preserving the expected defaults:
```sh
$ make -C cilium-cli install
  [...]
  install -m 0755 -d /usr/local/bin
  install -m 0755 cilium /usr/local/bin
```

---

Thanks @HadrienPatte That sounds like a much better approach. I have to admit that I need to educate myself on how the `Makefile.def` takes precedence, but it does fix the current issue. Thanks!